docs/dasharo-tools-suite/documentation.md

```diff
@@ -28,21 +28,24 @@ to be fixed before it is possible:
 The Dasharo Tools Suite can be started in various ways. Currently, there are
 two options:
 
-* bootable over network (iPXE),
+* bootable over a network (iPXE),
 * bootable USB stick image.
 
-The first one should be always preferred if possible, as it the easiest one to
-use.
+The first one should always be preferred if possible, as it is the easiest one
+to use.
 
-### Bootable over network
+### Bootable over a network
 
 This section describes how to boot DTS using iPXE.
 
 #### Requirements
 
-* Dasharo device with DTS functionality integrated
-* Wired network connection
-* [Secure Boot disabled](#disabling-secure-boot)
+Below are the requirements that must be met to run DTS over a network on the
+platform:
+
+* Dasharo device with DTS functionality integrated,
+* wired network connection,
+* [Secure Boot disabled](#disabling-secure-boot).
```

Comment:
@TomaszAIR and how about `Bios lock` (https://docs.dasharo.com/unified-test-documentation/dasharo-security/20J-bios-lock-support/). Is that feature should not set as disabled?

Reply:
@sulewskiprzemyslaw https://github.com/Dasharo/docs/pull/369#discussion_r1111653169

Reply:
@TomaszAIR OK and how about flashing procedures (I'm mainly concerned about the update). If the bios lock is turned on, won't it brick the platform? If so, we should add information about bios lock turning off in the dedicated section of the documentation.

Reply:
@sulewskiprzemyslaw do we have tests for updating platforms (flashing with flashrom) with lock turned on? Do we know what exactly can happen when we do such thing? Is this lock enabled by default on every platform? Only when Dasharo is there or on vendor firmware also?

If you feel that such note should be added, please suggest what exactly should we add and where.

Reply:
@TomaszAIR this functionality is quite new, but I remember, that we turn off this functionality to avoid flashing problems (otherwise, the platform may break down).

In my opinion, for now, the easiest option is to add information about this feature blockage in the requirements section.

Ultimately, in turn, each functionality described should have its own section with requirements.

Reply:
@TomaszAIR I just add a mention about disabling the BIOS lock feature (commit: https://github.com/Dasharo/docs/pull/369/commits/3b6ad2cca59cc5dbfb9a6478bded5840a9bef6ae)